package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rocket-grid-datatable",
-  "version": "1.0.0",
+  "version": "1.0.2",
```

Comment:
This PR makes a BC - new version will be `2.0.0`

+ change of the version should be separated from change to different commit/PR.